test/torch/tensors/test_precision.py

```diff
@@ -37,6 +37,13 @@ def test_fix_prec_registration(hook):
         assert hook.local_worker.get_obj(x.id) == x
 
 
+def test_fixed_precision_mod_operation():
```

Comment:
I expect that this modification will break the same functionality if the devisor was another FPT or torch.Tensor
please test the case of 
x = torch.tensor([1, 2, 3]).fix_prec()
y =  torch.tensor([3]).fix_prec()
z= (x%y).float_prec()
assert (z == torch.tensor([1.0, 2.0, 0.0])).all()

Reply:
That's a good shout. Will add that test too.

Reply:
I've added a test for this now.